tests/mentorship_relation/test_dao_cancel_relation.py

```diff
@@ -41,15 +41,15 @@ def setUp(self):
     def test_dao_cancel_non_existing_mentorship_request(self):
         DAO = MentorshipRelationDAO()
 
-        result = DAO.cancel_relation(self.first_user.id, 123)
+        result = DAO.cancel_relation(self.first_user.id, 123, 'Some reason to cancel the relation.')
```

Comment:
You'll have to fix these tests once you change how this function receives data

Reply:
Here you can already pass data json object with the string instead of just the string, in the 3º argument.